osu.Game/Screens/Multi/Match/MatchSubScreen.cs

```diff
@@ -44,6 +45,9 @@ public class MatchSubScreen : MultiplayerSubScreen
         [Resolved]
         private BeatmapManager beatmapManager { get; set; }
 
+        [Resolved]
+        private PreviewTrackManager trackManager { get; set; }
```

Comment:
seems kinda misleading name
```suggestion
        private PreviewTrackManager previewTrackManager { get; set; }
```